### PR TITLE
Izabel & Sofie's library project 

### PR DESCRIPTION
**Pull request by Izzibizz**
## Netlify link
https://helloworld-destinations.netlify.app

## Collaborators
Izabel Lind Färnstrand, Sofie Ferrari Strahl
[izzibizz, SofieFerrari]

### CODE REVIEW

File: style.css
Comment (NellaMori) [inline style.css:1]: Please format your code to have the right spaces and indentation 👍

**Review by NellaMori — APPROVED** [review]
Ladies!! I am _more_ than impressed! LOVE your project 😍 I mean...it's amazing! I have nothing more to add, you worked so beautifully, I am so happy when I see you have fun and get super creative with a project! Way to go 🚀 You followed the instructions, guidelines to write good code (just left a comment about formatting 1 file), made the app responsive and used filters properly combining them to the sorting. Well done, keep up the fantastic work 🥇🔥

File: script.js [hunk 1]
```diff
@@ -1,406 +1,292 @@
...
+}
+// Calling the function as soon as the website opens
+loadDestinations(destinations)
+
+//function that shift the value each time button is clicked, to enable toggle
+const tempToggle = () => {
+  lowToHigh = !lowToHigh
+  sortByTemp()
+}
+
```
Comment (Martin-Joensson) [inline script.js:209]: We love this, so simple and sweet! We will shamelessly steal this to our corrections. =)

/Martin & Mai

File: script.js [hunk 2]
```diff
@@ -1,406 +1,292 @@
...
+const dropDown = document.getElementById('filter-dropdown')
+const buttonRandom = document.getElementById('button-random')
+const searchInput = document.getElementById("search")
 
-const books = [
+//global scope
+let lowToHigh = true
+
+//Array with objects with different destinations
+const destinations = [
```
Comment (Martin-Joensson) [inline script.js:13]: Really like that you made you're own array of objects! Such a fun touch. 
/Martin & Mai

File: script.js [hunk 3]
```diff
@@ -1,406 +1,292 @@
...
+    })
+    loadDestinations(searchResult)
+  }
+  
+//EventListners 
+buttonRandom.addEventListener('click', filterRandom)
+buttonTemp.addEventListener('click', tempToggle)
+buttonRate.addEventListener('click', rateToggle)
+dropDown.addEventListener('change', filterCountry)
+searchInput.addEventListener('input', handleSearch)
```
Comment (maikanetaka) [inline script.js:291]: We think it is a great idea to have 'input' here instead of 'submit' so no need to click enter!
// Martin & Mai

**Review by Martin-Joensson — COMMENTED** [review]
As Antonella said, your site is super awesome! Great job guys. Took a lot of inspiration from your code! Be proud!

**Pull request closed without merging**